src/Pages/Admin/Admin.css

```diff
@@ -0,0 +1,15 @@
+.admin{
+    margin-top:80px;
+}
+.button-admin{
+    background-color:#D5D7CC;
+    border: none;
+    color: black;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
```

Comment:
Need a button like this(No shadows and also, button when clicked remains clicked, it should not retain it's clicked state).
![button](https://user-images.githubusercontent.com/56497513/114335496-83806300-9b6a-11eb-832c-4465e5e8c3d2.JPG)

You can use react-bootstrap for this.

Refer this : https://react-bootstrap.github.io/components/buttons/